content/en/spec/_index.md

```diff
@@ -6,6 +6,6 @@ date: 2022-09-15 20:55:05.533505515 +0100 IST m=+0.000179772
 draft: false
 ---
 
-| Version  | Description                              | Schema |
-| -------- | ---------------------------------------- | -------|
-| [v0.1.0 (Latest)](https://github.com/open-component-model/ocm-spec/tree/v0.1.0) | The most up-to-date version of the OCM specification | [v2](/schemas/component-descriptor-v2) ([Web Version](schema-v2.html)) \| [v3alpha1](/schemas/component-descriptor-v3alpha1) ([Web Version](schema-v3alpha1.html))
+The most up-to-date version of the [OCM specification](https://github.com/open-component-model/ocm-spec/blob/main/README.md) offers you a deep insight to all technical details of the Open Component Model and explains model OCM is based on.
+
+The available schema versions for OCM component descriptors are [v2](/schemas/component-descriptor-v2) ([Web Version](schema-v2.html)) and [v3alpha1](/schemas/component-descriptor-v3alpha1) ([Web Version](schema-v3alpha1.html)).
```

Comment:
what about only mentioning v3? see the other discussion on v2 vs. v3...

Reply:
@hilmarf, what about my last commit? With the discussion on another "version" for an sbom-like serialisation format this is for me the best solution currently.